### ISSUE TITLE
`default_limit` should not attach `_limit` param to a find request

### ISSUE THREAD

**Issue by Hkly** [beezwax/fmrest-ruby#35]
When I have a `default_limit` set, it attaches the `_limit` param to the GET request when I try to find by record id

```
https://some.fm.data.base/layouts/SomeDataLayout/records/465?_limit=999999
```

returns the error "Unknown parameter(s): _limit", error code "960"

**Comment by pilaf** [comment]
@Hkly I pushed a potential fix to the branch named discard-collection-params-on-find-single

Could you try using the gem from that branch and checking if it solves the problem for you?

Here's how to use that branch:

```ruby
gem "fmrest", github: "beezwax/fmrest-ruby", branch: "discard-collection-params-on-find-single"
```

Here's a pull request in case you want to review changes: https://github.com/beezwax/fmrest-ruby/pull/36

**Comment by Hkly** [comment]
Thanks @pilaf! Tested it and it works!

**Comment by pilaf** [comment]
Included in v0.5.0 release.
